Mockups/Base/MockCubeBlock.debug.cs

```diff
@@ -43,19 +42,11 @@ public abstract class MockCubeBlock : MockEntity, IMyCubeBlock
         public virtual Vector3I Position { get; set; }
 
         public virtual MyRelationsBetweenPlayerAndBlock GetUserRelationToOwner(long playerId)
-        {
-            throw new NotImplementedException();
-        }
+            => MyRelationsBetweenPlayerAndBlock.Neutral;
 
-        public virtual void UpdateIsWorking()
-        {
-            throw new NotImplementedException();
-        }
+        public virtual void UpdateIsWorking() { }
 
-        public virtual void UpdateVisual()
-        {
-            throw new NotImplementedException();
-        }
+        public virtual void UpdateVisual() { }
```

Comment:
These methods shouldn't be used by a script.

Reply:
Would make more sense to throw `NotSupportedException` or `InvalidOperationException` I guess?